### PR TITLE
Make Create Post

### PR DESCRIPTION
**Pull request by Can-Tina**

### CODE REVIEW

File: src/client/styling/createPost.css [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    align-items: center;
+    gap: 5px;
+    padding: 2px 10px 2px 10px
+}
+
+h1 {
+    text-align: center;
+    margin-top: 50px;
+    margin-bottom: 25px;
+    color: #2e2e2e;
```
Comment (TheunisK) [inline src/client/styling/createPost.css:51]: It all looks flawless Lenni... Love that my indenting the attributes finally rubbed off on you!

Comment (Can-Tina) [reply src/client/styling/createPost.css:51]: I was reminded of you while doing it :)

**Comment by nathanSamosa** [comment]
All looks good to me :)

**Comment by bravint** [comment]
I approve

**Comment by vherus** [comment]
Your code is nice and clean but the commit message could use some improvement: "Make Create Post" out of context is difficult to understand.

Try to stick to words like "add, remove, change" - `add CreatePost component` would be easier to understand here.

**Review by vherus — APPROVED** [review]

**Pull request merged**